src/components/OptionsList/BaseOptionsList.tsx

```diff
@@ -133,7 +135,8 @@ function BaseOptionsList(
      *
      *     [{header}, {sectionHeader}, {item}, {item}, {sectionHeader}, {item}, {item}, {footer}]
      */
-    const getItemLayout = (_data: OptionsListData[] | null, flatDataArrayIndex: number) => {
+    // eslint-disable-next-line @typescript-eslint/naming-convention
+    const getItemLayout = (_data: OptionsListDataWithIndexOffset[] | null, flatDataArrayIndex: number) => {
```

Comment:
Is there a reason we need to violation this lint rule, or can we do something like this?
```suggestion
    const getItemLayout = (data: OptionsListDataWithIndexOffset[] | null, flatDataArrayIndex: number) => {
```